pkg/apis/monitoring/v1/types.go

```diff
@@ -79,7 +79,7 @@ type PrometheusRuleExcludeConfig struct {
 type ProxyConfig struct {
 	// `proxyURL` defines the HTTP proxy server to use.
 	//
-	// It requires Prometheus >= v2.43.0.
+	// It requires Prometheus >= v0.16.0.
```

Comment:
seems like an unwanted change?

Reply:
From what I can see `proxyURL` seems to be supported above `v0.16.0` could you verify?